Cargo.toml

```
@@ -43,6 +43,7 @@ regex = { version = "1.9", default-features = false }
 rustc-hash = { version = "2.1", default-features = false, features = [ "std" ]}
 static_assertions = { version = "1.1", default-features = false }
 strum = { version = "0.27", default-features = false, features = ["derive"] }
+tempfile = { version = "3.16.0", default-features = false }
```

```suggestion
tempfile = { version = "3.16", default-features = false }
```
Might be sufficient here to use this, since `Cargo.lock` then locks it to the version used in Firefox.